nservicebus/messaging/headers.md

```diff
@@ -322,19 +322,11 @@ The NServiceBus version number.
 
 ## OpenTelemetry-related headers
 
-These headers are added when [OpenTelemetry](/nservicebus/operations/opentelemetry.md) is enabled on the endpoint.
+These headers are added when [OpenTelemetry](/nservicebus/operations/opentelemetry.md) is enabled for an endpoint:
```

Comment:
```suggestion
These headers are added when [OpenTelemetry](/nservicebus/operations/opentelemetry.md) is enabled for an endpoint, in accordance with [W3C TraceContext specification](https://www.w3.org/TR/trace-context):
```

I think adding my suggestion may prevent users from having to always click through to understand why these are added.

Reply:
👍 small tweaks:

```suggestion
These headers are added when [OpenTelemetry](/nservicebus/operations/opentelemetry.md) is enabled for an endpoint, in accordance with the [W3C Trace Context specification](https://www.w3.org/TR/trace-context):
```